main/src/ca/uwaterloo/flix/tools/pkg/ManifestParser.scala

```diff
@@ -16,22 +16,218 @@
 package ca.uwaterloo.flix.tools.pkg
 
 import ca.uwaterloo.flix.util.Result
-import org.tomlj.Toml
+import ca.uwaterloo.flix.util.Result.{Err, Ok}
+import org.tomlj.{Toml, TomlArray, TomlInvalidTypeException, TomlParseResult, TomlTable}
 
-import java.io.{PrintStream, StringReader}
-import java.nio.file.Path
+import java.io.{IOException, PrintStream}
+import java.nio.file.{InvalidPathException, Path, Paths}
+import scala.collection.mutable
 
 object ManifestParser {
 
-  def parse(p: Path)(implicit out: PrintStream): Result[Manifest, ManifestError] = ???
```

Comment:
What should the `out` stream be used for?

Reply:
I don't think we need it here. For some other stuff its to report progress (println) while the algorithm is running.